source/newcommitter.md

```diff
@@ -95,12 +95,12 @@ If they accept, then do:
 <a name="NewCommitter-Discussion"></a>
 ### Discussion
 
-We do the vote on the private mailing list to enable a frank discussion.
+We do the vote on the `private@` mailing list to enable a frank discussion.
```

Comment:
If you want to make it clear what the mail list is called, perhaps private@ is a better term than @private.